service/http.directory/src/test/scripts/org/kaazing/gateway/service/http/directory/file.symlink.returns.200.rpt

```diff
@@ -0,0 +1,7 @@
+connect http://localhost:8701/link
```

Comment:
its not clear from this test, but the "link" is to a file or a directory, maybe rename link to link-to-file